R/aaa-dbplyr.R

```diff
@@ -1,4 +1,8 @@
 replace_if_dbplyr_has <- function(fun) {
+  if (!requireNamespace("dbplyr", quietly = TRUE)) {
+    return()
```

Comment:
`return(invisible())` ?

Reply:
You're right, but it's not too important here and below, because this is an internal function called in `.onLoad()` .